locales/en-US/app.ftl

```diff
@@ -12,9 +12,17 @@ footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkLegal = Legal
 footerLinkFeedback = Give Feedback
-footerLinkAbout = About Test Pilot
-
-## Items in the menu.
+footerLinkAboutHeader = About
```

Comment:
After looking at the code, I think it would be a good idea to add a comment to this one, e.g.

```
# This is a header displayed above a set of links about Mozilla and Test Pilot
```